editor/src/components/editor/global-shortcuts.ts

```diff
@@ -812,8 +812,10 @@ export function handleKeyUp(
   const editorTargeted = editorIsTarget(event, editor)
   let updatedKeysPressed: KeysPressed
   if (editorTargeted) {
```

Comment:
If the code here makes sense, there's no longer a need for this condition here, since both branches of the `if` statement do the same thing...

Reply:
This looks correct, but I'm not sure of the reasoning why it was done like this in the first place? @seanparsons do you recall?

Reply:
I think this was done this way to mirror what is in `handleKeyDown`, I don't immediately see any reason why we shouldn't collapse it for key up, as it looks wrong now in the cold light of day.

Reply:
I guess we'll discover if there truly _was_ a reason for this, but the reference to editorTargeted makes me think it's from draft days.

Reply:
Quite possibly yeah. Worst case scenario we break the editor, but that's really _worst case_ so it's probs fine